assets/stylesheets/_sections.scss

```diff
@@ -21,15 +26,21 @@
 }
 
 .hero-section {
-  background: url("#{$asset-path}" + "#{$hero-background}"), no-repeat;
+  background: 
+    black 
+    url("#{$asset-path}" + "#{$hero-background}");
+  background-repeat: no-repeat;
   background-size: cover;
   padding: 6em 0;
   text-align: center;
+
 }
 
 .hero-section__logo {
   margin: 0 auto;
   margin-bottom: 4em;
+  width:100%;
+  max-width: 600px;
```

Comment:
🤔 is this implementation-specific? if not, would it make sense to extract this to `$width--med`? (`$wrapper--med`, etc…?)